src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -14,159 +14,133 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-        <tag name="LOG" description="Logging" />
-        <tag name="WDSCH" description="Week Day Access Schedules" />
-        <tag name="DPS" description="Door Position Sensor" />
-        <tag name="FACE" description="Face Credentials" />
-        <tag name="COTA" description="Credential Over-the-Air Access" />
-        <tag name="USR" description="User" />
-        <tag name="NOT" description="Notification" />
-        <tag name="YDSCH" description="Year Day Access Schedules" />
-        <tag name="HDSCH" description="Holiday Schedules" />
-        <tag name="UBOLT" description="Unbolting" />
-        <tag name="ALIRO" description="Aliro Credential Provisioning" />
-
-        <client tick="false" init="false">true</client>
-        <server tick="false" init="false">true</server>
-
-        <!-- Current cluster version -->
-        <globalAttribute side="either" code="0xFFFD" value="7" />
-        <!-- Cluster feature map -->
-        <globalAttribute side="server" code="0xFFFC" value="0x0001">
-            <featureBit tag="PIN" bit="0">true</featureBit>
-            <featureBit tag="RID" bit="1">false</featureBit>
-            <featureBit tag="FGP" bit="2">false</featureBit>
-            <featureBit tag="LOG" bit="3">false</featureBit>
-            <featureBit tag="WDSCH" bit="4">true</featureBit>
-            <featureBit tag="DPS" bit="5">true</featureBit>
-            <featureBit tag="FACE" bit="6">false</featureBit>
-            <featureBit tag="COTA" bit="7">true</featureBit>
-            <featureBit tag="USR" bit="8">true</featureBit> <!-- TODO: Depends on ALIRO, [ PIN | RID | FGP | FACE ]-->
-            <featureBit tag="NOT" bit="9">false</featureBit>
-            <featureBit tag="YDSCH" bit="10">true</featureBit>
-            <featureBit tag="HDSCH" bit="11">true</featureBit>
-            <featureBit tag="UBOLT" bit="12">true</featureBit>
-            <featureBit tag="ALIRO" bit="13">true</featureBit>
-            <featureBit tag="ALBU" bit="14">true</featureBit> <!-- TODO: Depends on ALIRO -->
-        </globalAttribute>
-
-        <features>
-            <feature bit="0" code="PIN" name="PIN Credential" summary="Lock supports PIN credentials (via keypad, or over-the-air)">
-            <optionalConform/>
-            </feature>
-            <feature bit="1" code="RID" name="RFID Credential" summary="Lock supports RFID credentials">
-            <optionalConform/>
-            </feature>
-            <feature bit="2" code="FGP" name="FingerCredentials" summary="Lock supports finger related credentials (fingerprint, finger vein)">
-            <otherwiseConform>
-                <provisionalConform/>
-                <optionalConform/>
-            </otherwiseConform>
-            </feature>
-            <feature bit="3" code="LOG" name="Logging" summary="Lock supports local/on-lock logging when Events are not supported">
-            <optionalConform/>
-            </feature>
```

Comment:
Enclosed in Zigbee conditionals in the spec definition, but its already included in the matter's xml version, so should be included here as well.
https://github.com/project-chip/connectedhomeip/blob/master/data_model/master/clusters/DoorLock.xml#L69